src/current/cockroachcloud/authorization.md

```diff
@@ -108,15 +117,45 @@ Cluster Administrators can perform all of the [Cluster Operator actions](#cluste
 - Cluster Administrators for the whole organization (rather than scoped to a single cluster) can [create new clusters](create-your-cluster.html).
 - Access the [DB Console](network-authorization.html#db-console).
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
 ### Cluster Creator
 
 Cluster Creators can create clusters in an organization. A cluster's creator is automatically granted the [Cluster Administrator](#cluster-administrator) role for that cluster upon creation.
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
```

Comment:
nit: Cluster Creator on a cluster has no utility so maybe remove the individual cluster part

Reply:
Done